### PR TITLE
modify location

### PR DESCRIPTION
**Pull request by ccfromms**
modify location returned from batch post

### CODE REVIEW

File: Service/Microsoft.Health.DeIdentification.Web/RouteNames.cs [hunk 1]
```diff
@@ -5,10 +5,12 @@
 
 namespace Microsoft.Health.DeIdentification.Web
 {
-    internal class RouteNames
+    internal static class RouteNames
```
Comment (hongyh13) [inline Service/Microsoft.Health.DeIdentification.Web/RouteNames.cs:8]: maybe we can remove this static class, and generate URL directly?

Comment (ccfromms) [reply Service/Microsoft.Health.DeIdentification.Web/RouteNames.cs:8]: I will modify it

File: Service/Microsoft.Health.DeIdentification.Web/Controllers/FhirController.cs [hunk 2]
```diff
@@ -107,7 +111,7 @@ public async Task<IActionResult> GetDeIdentificationJobStatus(string operationId
             }
             else
             {
-                string url = GenerateUrl(operationId);
+                string url = $"{Request.HttpContext.Request.Scheme}://{host}/base/operation/{operationId}";
```
Comment (hongyh13) [inline Service/Microsoft.Health.DeIdentification.Web/Controllers/FhirController.cs:114]: we can use a GenerateUrl function with request and operation URL passed as parameters.

**Review by hongyh13 — APPROVED** [review]

File: Service/Microsoft.Health.DeIdentification.Web/Controllers/FhirController.cs (hunk 2 above)
Comment (ccfromms) [reply Service/Microsoft.Health.DeIdentification.Web/Controllers/FhirController.cs:114]: I will restore it

**Pull request merged**